Changelog.md

```diff
@@ -1,5 +1,85 @@
 ## libsdformat 10.X
 
+### libsdformat 10.6.0 (2021-09-08)
+
+1. Parse URDF continuous joint effort/velocity limits
+    * [Pull request #684](https://github.com/ignitionrobotics/sdformat/pull/684)
+
+1. Add enable_orientation SDF element to imu
+    * [Pull request #651](https://github.com/ignitionrobotics/sdformat/pull/651)
+
+1. Add a codecheck make target
+    * [Pull request #682](https://github.com/ignitionrobotics/sdformat/pull/682)
+
+1. Refactor sdf::readXml
+    * [Pull request #681](https://github.com/ignitionrobotics/sdformat/pull/681)
+
+1. Upgrade cpplint and fix new errors
+    * [Pull request #680](https://github.com/ignitionrobotics/sdformat/pull/680)
+
+1. BUG: add missing plugin element to include
+    * [Pull request #675](https://github.com/ignitionrobotics/sdformat/pull/675)
+
+1. Update codecov badge url
+    * [Pull request #678](https://github.com/ignitionrobotics/sdformat/pull/678)
+
+1. Added comment reminder to update functions
+    * [Pull request #677](https://github.com/ignitionrobotics/sdformat/pull/677)
+
+1. Port to new codecov configuration
+    * [Pull request #676](https://github.com/ignitionrobotics/sdformat/pull/676)
+
+1. Add pull request triage to sdf9
+    * [Pull request #673](https://github.com/ignitionrobotics/sdformat/pull/673)
+
+1. Adds enable_metrics flag to Sensor.
+    * [Pull request #665](https://github.com/ignitionrobotics/sdformat/pull/665)
+
+1. Add GPS sensor to sdf9
```

Comment:
the commit message will always say GPS, but should we change `GPS` to `NavSat` to match the renaming applied in the code?